build: use whiskers to generate themes (#2)
catppuccin/halloy: README.md

## Usage

> 1. Download your flavor of choice from [`themes/`](./themes/) to halloy's themes directory:
>     - Linux: `$HOME/.config/halloy/themes`
>     - macOS: `$HOME/Library/Application Support/halloy/themes`
>     - Windows: `{FOLDERID_RoamingAppData}\halloy\themes`

Reviewer: ```suggestion
1. Download your flavor of choice from [`themes/`](./themes/) to the `themes/` directory inside [Halloy's configuration directory](https://halloy.squidowl.org/configuration/index.html).
```

> 2. Update the `theme` field of your `config.toml` to the name of the theme without the `.toml` file extension:
>     - `theme = "catppuccin-frappe"`

Reviewer: ```suggestion
2. Update the `theme` field of your `config.toml` to `catppuccin-<flavor>`, where `<flavor>` is your chosen flavor. For example, `theme = "catppuccin-frappe"`.
```

> 3. Restart Halloy

Reviewer: ```suggestion
3. Restart Halloy to see the theme applied.
```
